frontend/src/metabase/browse/components/CollectionBreadcrumbsWithTooltip.tsx

```diff
@@ -85,52 +86,62 @@ export const CollectionBreadcrumbsWithTooltip = ({
         name={containerName}
         w="auto"
       >
-        <Flex align="center" w="100%" lh="1" style={{ flexFlow: "row nowrap" }}>
-          <CollectionsIcon
-            name="folder"
-            // Stopping propagation so that the parent <tr>'s onclick won't fire
-            onClick={(e: React.MouseEvent) => e.stopPropagation()}
-          />
-          {shownCollections.map((collection, index) => {
-            const key = `collection${collection.id}`;
-            return (
-              <BreadcrumbGroup
-                spacing={0}
-                key={key}
-                // Stopping propagation so that the parent <tr>'s onclick won't fire
-                onClick={(e: React.MouseEvent) => e.stopPropagation()}
-              >
-                {index > 0 && <PathSeparator />}
-                <CollectionBreadcrumbsWrapper
-                  containerName={containerName}
-                  style={{ alignItems: "center" }}
-                  w="auto"
-                  display="flex"
+        <CollectionLink to={Urls.collection(collection)}>
```

Comment:
For some reason this link does a full navigation. Nick and I studied the problem and we didn't figure it out. I don't think it's worth putting any more time into solving it.